### PR TITLE
fix(rds): switch connection monitor to average

### LINKED ISSUE #231: [rds] Connection monitor should use AVERAGE not SUM
### Version

1.22.0

### Steps and/or minimal code example to reproduce

```typescript
new MonitoringFacade(this, 'Monitoring', {}).monitorRdsCluster({ cluster });
```

### Expected behavior

Connection count monitor should be an average (not sum) for readability and to match the default RDS monitor dashboard

### Actual behavior

Its using SUM which leads to a confusing over count of connections

### Other details

_No response_

### PR DESCRIPTION
**Pull request by relm923**
Fixes #231 

---

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license_

### CODE REVIEW

**Review by echeung-amzn — APPROVED** [review]
LGTM; will wait for another approval before merging.

**Comment by relm923** [comment]
🤔 this is more accurate but ideally it should be creating a separate monitor per instance

**Review by ayush987goyal — APPROVED** [review]

**Pull request merged**
